### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast : The "ms-azuretools.vscode-azurefunctions" text has color contrast ratio less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2280]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#E2E-AzureExtensionsForVSCode-Mac-Aug2020;#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#FTP;#Benchmark;#ColorContrast;#A11ySev2:#WCAG1.4.3;#External: VSCode;

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Navigate to "ms-azuretools.vscode-azurefunctions" text and verify if the color contrast ratio is less than 4.5:1 or not.

### Actual:
The "ms-azuretools.vscode-azurefunctions" text has color contrast ratio of 2.6:1 which is less than the standard color contrast ratio of 4.5:1 which is not appropriate as the users with low vision will have difficulty in reading the text on the screen.

### Expected:
The color contrast ratio of "ms-azuretools.vscode-azurefunctions" text should be greater than or equal to 4.5:1.

### User Impact:
If the color contrast ratio of the content present on the screen is less than 4.5:1 then the users with low vision will have difficulty in reading the data on the screen.

### Attachment link for Reference:
<img width="1400" alt="2280_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast The  ms-azuretools vscode-azurefunctions  text has color contrast ratio less than 4 51" src="https://user-images.githubusercontent.com/52036772/89617823-8541aa80-d8a8-11ea-9f3d-b93149881fa7.png">

**Comment by EricJizbaMSFT** [comment]
This issue should be filed in the VS Code repo: https://github.com/Microsoft/vscode

Our extension has no control over that section of the marketplace page. For example, if you look at any extension in the marketplace, it has the exact same styling and behavior:
![Screen Shot 2020-08-10 at 11 09 01 AM](https://user-images.githubusercontent.com/11282622/89815612-ead8a580-daf9-11ea-9e13-6b6b94476989.png)

**Comment by nirmalsahoo** [comment]
Logged bug in VS Code-  https://github.com/microsoft/vscode/issues/104551
As per the C & AI bug process assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by MayuriNaik** [comment]
Logged an external bug to VS Code: [108300](https://github.com/microsoft/vscode/issues/108300)

**Comment by EricJizbaMSFT** [comment]
This has been fixed in VS Code Insiders.

![Screen Shot 2020-11-17 at 11 29 41 AM](https://user-images.githubusercontent.com/11282622/99441083-4326fc80-28cc-11eb-8b19-74267b070860.png)

Closing for that reason and due to lack of response from accessibility test team.
